sdk/openai/Azure.AI.OpenAI/tests/Samples/SpeechGeneration.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Identity;
+using NUnit.Framework;
+
+namespace Azure.AI.OpenAI.Tests.Samples;
+
+public partial class SpeechGeneration
+{
+    [Test]
+    [Ignore("Only verifying that the sample builds")]
+    public async Task TranscribeAudio()
```

Comment:
Update test method name

Reply:
Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/11/commits/07347328bfaac7fb3dbc92bcf5b93145eb0e7a04